admin/controllers/check-window.js

```diff
@@ -60,9 +62,15 @@ const controller = {
    */
   submitCheckWindow: async (req, res, next) => {
     const requestData = req.body
+    const { checkWindowUrlSlug } = requestData
     let flashMessage
     try {
-      flashMessage = await checkWindowV2AddService.submit(requestData)
+      if (checkWindowUrlSlug) {
+        flashMessage = await checkWindowV2UpdateService.submit(requestData)
```

Comment:
This seems odd to me.  Why is a service returning a message that seems to be targeted at the GUI?

Reply:
@GeorgeChatzigiannis might be better to just fail/succeed the service via a try/catch call and curate an appropriate flash message in the controller

Reply:
@GuyHarwood @jon-shipley this is now updated